<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-KPO36PO" failures="2" tests="2" name="Test(failed)" time="13.938" errors="0" timestamp="2024-09-24T14:48:44 IST">
  <testcase classname="testCases.TC03_Login_DTT_Test" name="verify_Login_DTT" time="2.441">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at testCases.TC03_Login_DTT_Test.verify_Login_DTT(TC03_Login_DTT_Test.java:64)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- verify_Login_DTT -->
  <testcase classname="testCases.TC03_Login_DTT_Test" name="verify_Login_DTT" time="1.886">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at testCases.TC03_Login_DTT_Test.verify_Login_DTT(TC03_Login_DTT_Test.java:64)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- verify_Login_DTT -->
</testsuite> <!-- Test(failed) -->
